extensions/azurePublishNew/src/components/provisioningWizards/HandOffToAdminWizard.tsx

```diff
@@ -35,16 +85,54 @@ export const HandOffToAdminWizard = React.memo((props: Props) => {
         onBack,
         onCancel,
       },
+      {
+        id: 'add-resources',
+        title: formatMessage('Add resources'),
+        subTitle: formatMessage.rich(
+          'Your bot needs the following resources based on its capabilities. Select resources that you want to provision in your publishing profile. <a>Learn more</a>',
+          {
+            a: ({ children }) => (
+              <a key="add-resource-learn-more" href={urls.createNewResources} rel="noopener noreferrer" target="_blank">
+                {children}
+              </a>
+            ),
+          }
+        ),
+        onRenderContent: () => (
+          <ChooseResourcesStep enabledResources={resources} onChangeSelection={setEnabledHandOffResources} />
+        ),
+        onCancel,
+        onNext: () => setShowHandOff(true),
+      },
     ]);
-  }, []);
+  }, [resources]);
 
   return (
-    <Wizard
-      firstStepId="handoff-instructions"
-      steps={steps}
-      onRenderFooter={(navState) => <WizardFooter {...navState} />}
-      onRenderHeader={(step) => setDialogTitle(step)}
-      onStepChange={(index, step) => onStepChange(index, step.id)}
-    />
+    <>
+      <Wizard
+        firstStepId="handoff-instructions"
+        steps={steps}
+        onRenderFooter={(navState) => <WizardFooter {...navState} />}
+        onRenderHeader={() => <></>}
+        onStepChange={(index, step) => onStepChange(index, step)}
+      />
+      {showHandOff && (
+        <ProvisionHandoff
+          developerInstructions={formatMessage(
+            'If Azure resources and subscription are managed by others, use the following information to request creation of the resources that you need to build and run your bot.'
+          )}
+          handoffInstructions={handOffInstructions}
+          hidden={!showHandOff}
+          learnMoreLink="https://aka.ms/how-to-complete-provision-handoff"
```

Comment:
add this to your urls object up, then it would make sense to have it as an object.